css/src/trojsten/protocol.less

```diff
@@ -0,0 +1,67 @@
+.protocol-sets {
+    overflow-x: scroll;
+}
+
+.protocol-set {
+    display: flex;
+    flex-wrap: nowrap;
+}
+
+@media (min-width: @screen-md-min) {
+    .protocol-set {
+        flex-wrap: wrap;
+    }
+}
+
+.protocol-result {
+    font-weight: bold;
+}
+
+.protocol-time {
+    font-size: 0.8em;
+}
+
+.protocol-set-name {
+    font-size: 0.9em;
+}
+
+.protocol-set-result {
+    font-size: 1em;
+}
+
+.protocol-box {
+    flex-shrink: 0;
+    width: 50px;
+    height: 50px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+
+    border: 1px solid darken(#fff, 5%);
+    background: #fff;
+}
+
+.protocol-ok {
+    background: @state-success-bg;
+    border: 1px solid darken(@state-success-bg, 5%);
```

Comment:
Tu staci len farba, zvysok sa moze pouzit z protocol-box. Este lepsie by mohlo byt pouzti 

```css
.protocol-box.bg-success {
  border-color: darken(@state-success-bg, 5%);
}
```
pripadne sa toho ramceka zbavit a mat pokoj, podla mna to aj tak vyzera dobre:
![FireShot Capture 001 - Detaily submitu - Korešpondenčný seminár z programovania - localhost](https://user-images.githubusercontent.com/1177655/67598558-b4e25480-f765-11e9-83a4-9e4d72d5c67c.png)

Reply:
`.bg-success` vyzerá, že nie je súčasťou BS3

Reply:
https://getbootstrap.com/docs/3.3/css/#helper-classes-backgrounds

Reply:
Oh, poradie CSSiek :D